packages/storybook/stories/tooltip.mdx

```diff
@@ -0,0 +1,46 @@
+import { Tooltip } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta } from '@storybook/blocks';
+import * as TooltipStories  from './tooltip.stories';
+
+# Tooltip
+
+## Table of contents
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+<Meta of={TooltipStories} />
+
+## Definition
+A tooltip provides extra information to the user. It becomes visible when a focusable element is hovered over or when it enters its focused state [1].
+<Canvas of={TooltipStories.Default} />
+
+## Usage
+
+### When to use:
```

Comment:
Plusieurs points disent sensiblement la même chose.
Je crois que tu pourrais regrouper les 3 premiers.
> Use to help users understand the meaning or the purpose of a focusable/interactive element by providing useful, additional information or clarification.

le bouton icône et le copy-to-clipboard (qui doit être un bouton) sont pas mal inclus dans la phrase.